### PR TITLE
Add link to refresh data dictionary

### PR DESCRIPTION
**Pull request by czue**
## Summary

Struggled to figure this out today - turns out it's a magic link in confluence docs. Maybe easier if it were more discoverable?

image

This PR just adds the grey text at the bottom of the screenshot.

I understand this could be a computationally expensive operation so may have intentionally been obfuscated. My thinking was that it might be ok since it's feature-flagged?

## Feature Flag

Data Dictionary

## Safety Assurance

- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [ ] I am certain that this PR will not introduce a regression for the reasons below

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Review by snopoke — COMMENTED** [review]
@dimagi/product can weigh in on the question of if this should be included or not.

:white_check_mark: from a code review point of view

**Comment by czue** [comment]
@snopoke Do you think @dimagi/product understands the implications of a large rebuild in terms of the possible impact on our infrastructure, etc.? I had pinged you so that you could weigh in from a sitre reliability perspective - not sure whether your :white_check_mark: includes that perspective.

Also, cc @dannyroberts

**Comment by snopoke** [comment]
> @snopoke Do you think @dimagi/product understands the implications of a large rebuild in terms of the possible impact on our infrastructure, etc.? I had pinged you so that you could weigh in from a sitre reliability perspective - not sure whether your  includes that perspective.

So this will:
- load all current apps to get all case types and case properties
- query ES for all case types
- load all app builds to get historical case types and properties
- save / update the SQL models

While that is expensive I wouldn't expect it to be done often. Perhaps we could put a limit on it so that it can't be done more than once per day per domain or something? Another option would be to further hide this under the "support" flag.

File: corehq/apps/data_dictionary/templates/data_dictionary/base.html
Comment (orangejenny) [inline corehq/apps/data_dictionary/templates/data_dictionary/base.html:100]: `class="help-block"` does greyed out text and top and bottom margins

Comment (czue) [reply corehq/apps/data_dictionary/templates/data_dictionary/base.html]: Ah - perfect! Thanks. 

https://github.com/dimagi/commcare-hq/pull/29042/commits/b77fd92678699227e5979a8875dbd1e4097ab274

**Comment by czue** [comment]
@snopoke @orangejenny @dimagi/product is it clear what the path to getting this either approved or closed is?

**Review by snopoke — APPROVED** [review]

**Comment by snopoke** [comment]
> @snopoke @orangejenny @dimagi/product is it clear what the path to getting this either approved or closed is?

Since this is under a solutions FF I'm OK to merge it as is.

**Comment by czue** [comment]
cool, thanks. Will wait till EOD boston time in case any other feedback and will merge tomorrow morning if no blockers.

**Comment by orangejenny** [comment]
Seems fine to me technically. @AndreaMKing5 is in charge of solutions UI changes.

**Comment by AndreaMKing5** [comment]
Is this being added to the "General toggle for support features" flag?  If so, that's under "Internal Engineering Tools" so Solutions doesn't have any particularly vested interest.  If we could add documentation [here ](https://confluence.dimagi.com/pages/viewpage.action?spaceKey=ccinternal&title=Support+Flag)to make it more discoverable that would be great.

**Comment by millerdev** [comment]
+1 to adding rate limit. I don't think we want someone to rage-click that.

**Comment by czue** [comment]
@snopoke @millerdev added rate limiting in https://github.com/dimagi/commcare-hq/pull/29042/commits/ff38fc330f8e9c86c8c6132c8661094adaaa29b9

**Comment by czue** [comment]
@AndreaMKing5 it's being added as GA to anyone using Data dictionary. I'll update DD docs once it's merged

**Review by snopoke — APPROVED** [review]

**Comment by snopoke** [comment]
@czue thanks for the rate limiting addition

**Pull request merged**
